### PR TITLE
Update plan-auto-attendant-call-queue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Consolidating licensing and other requirements into the Prerequisites section of this page.  Once completed and published, will update other pages to reference this.  This should prevent the currently occuring problem where licensing requirements across multiple pages do not match.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/plan-auto-attendant-call-queue.md [hunk 1]
```diff
@@ -65,11 +65,20 @@ To configure auto attendants and call queues, you need the following resources:
...
 - A free Phone System - Virtual User license for each resource account
-- A service number for each auto attendant or call queue that you want to be directly dialable
-- A calling plan for each person who will be receiving call queue calls
+- At least one [Microsoft service number](getting-service-phone-numbers.md), direct routing number, or a hybrid number for each resource account that you want to be directly dialable
+ - The service number may be a toll or toll-free number
+
+Agents who receive calls from the call queues must be Enterprise Voice enabled online or on-premise users. 
+<br>If your agents are using the Microsoft Teams app for call queue calls, they need to be in TeamsOnly mode.
+
+When transferring calls to an external phone number, the resource account performing the transfer (ie. the one associated with the auto attendant or call queue) must have a phone number and a Microsoft 365 Phone System - Virtual User license. Additionally:
```
Comment (JohanFreelancer9) [inline Teams/plan-auto-attendant-call-queue.md:74]:
```suggestion
When transferring calls to an external phone number, the resource account performing the transfer (that is, the one associated with the auto attendant or call queue) must have a phone number and a Microsoft 365 Phone System Virtual User license. Additionally:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**